tests/Unit/ApparentHorizons/Test_ApparentHorizonFinder.cpp

```diff
@@ -0,0 +1,350 @@
+  using interpolator_source_vars =
+      tmpl::list<gr::Tags::SpacetimeMetric<3, Frame::Inertial>,
+                 GeneralizedHarmonic::Pi<3, Frame::Inertial>,
+                 GeneralizedHarmonic::Phi<3, Frame::Inertial>>;
+  using interpolation_target_tags = tmpl::list<AhA>;
+  using temporal_id = Time;
+  using domain_frame = Frame::Inertial;
+  static constexpr size_t domain_dim = 3;
+  using component_list =
+      tmpl::list<mock_interpolation_target<MockMetavariables, AhA>,
+                 mock_interpolator<MockMetavariables>>;
+  using const_global_cache_tag_list = tmpl::list<>;
+  enum class Phase { Initialize, Exit };
+};
+
+template <typename PostHorizonFindCallback>
+void test_apparent_horizon(const gsl::not_null<size_t*> test_horizon_called,
+                           const size_t l_max,
+                           const size_t grid_points_each_dimension,
+                           const double mass,
+                           const std::array<double, 3>& dimensionless_spin) {
+  using metavars = MockMetavariables<PostHorizonFindCallback>;
+  using MockRuntimeSystem = ActionTesting::MockRuntimeSystem<metavars>;
+  using TupleOfMockDistributedObjects =
+      typename MockRuntimeSystem::TupleOfMockDistributedObjects;
+  TupleOfMockDistributedObjects dist_objects{};
+  using MockDistributedObjectsTagTargetA =
+      typename MockRuntimeSystem::template MockDistributedObjectsTag<
+          mock_interpolation_target<metavars, typename metavars::AhA>>;
+  using MockDistributedObjectsTagInterpolator =
+      typename MockRuntimeSystem::template MockDistributedObjectsTag<
+          mock_interpolator<metavars>>;
+  tuples::get<MockDistributedObjectsTagTargetA>(dist_objects)
+      .emplace(
+          0,
+          ActionTesting::MockDistributedObject<
+              mock_interpolation_target<metavars, typename metavars::AhA>>{});
+  tuples::get<MockDistributedObjectsTagInterpolator>(dist_objects)
+      .emplace(
+          0,
+          ActionTesting::MockDistributedObject<mock_interpolator<metavars>>{});
+
+  // Options for all InterpolationTargets.
+  intrp::OptionHolders::ApparentHorizon<Frame::Inertial> apparent_horizon_opts(
+      Strahlkorper<Frame::Inertial>{l_max, 2.8, {{0.0, 0.0, 0.0}}}, FastFlow{},
+      Verbosity::Verbose);
+
+  tuples::TaggedTuple<typename metavars::AhA> tuple_of_opts(
+      apparent_horizon_opts);
+
+  MockRuntimeSystem runner{tuple_of_opts, std::move(dist_objects)};
+
+  const auto domain_creator = domain::creators::Shell<Frame::Inertial>(
+      1.9, 2.9, 1, {{grid_points_each_dimension, grid_points_each_dimension}},
+      false);
+
+  runner.template simple_action<
+      mock_interpolation_target<metavars, typename metavars::AhA>,
+      ::intrp::Actions::InitializeInterpolationTarget<typename metavars::AhA>>(
+      0, domain_creator.create_domain());
+  runner.template simple_action<mock_interpolator<metavars>,
+                                ::intrp::Actions::InitializeInterpolator>(0);
```

Comment:
Maybe this should be an issue.  It is done with a literal `0` in all other tests, so we should be consistent.

Reply:
In this case I think consistency with other code is not important and you can just make it clearer in this test. It's a minor thing, but I think it can help.